freedomofpress/securedrop-workstation
# Uses tags for RPC grants denoting Client privileges

Updates the Qubes RPC policy definitions to use a tag `sd-client` rather than hardcoded VM name `sd-svs`, in order to accommodate dev envs on Qubes for working on the [SecureDrop Client](https://github.com/freedomofpress/securedrop-client). Related issues:

* Closes #288.
* Refs https://github.com/freedomofpress/securedrop-client/pull/490
* Refs https://github.com/freedomofpress/securedrop-debian-packaging/pull/72

### Testing

0. In dom0, run `make clone && make clean && make all`
1. Confirm that `make test` passes without errors
2. Run `qvm-tags sd-dev add sd-client`
3. Proceed with review of https://github.com/freedomofpress/securedrop-client/pull/490 to confirm new tagging setup permits `sd-dev` to open Client submissions in DispVMs
